client/src/views/FlightEdit.vue

```diff
@@ -18,6 +18,7 @@
     </div>
     <!-- TODO: Admin edit shows wrong gliders -->
     <GliderSelect
+      v-if="getUserId == flight.userId"
```

Comment:
Hm... würde ich hier tatsächlich nicht rausnehmen, sondern mir aus dem Backend die konkreten Glider des Users und nicht die des Admin senden lassen.

So besteht die Möglichkeit, dass ein Admin auch nach 14 Tagen einen Flug korrigieren kann.

Reply:
Hatte ich zuerst überlegt, das zieht allerdings weitreichende Änderungen in Back und Frontend mit sich, daher dachte ich, ich entferne es erstmal, damit die Seite überhaupt nutzbar ist. Dann kann man immer noch schaue, ob man es einbaut, oder für den extrem unwahrscheinlichen Fall, dass man tatsächlich mal den glider wechseln wollen sollte, könnte man es immer noch in der DB machen.

Reply:
Habe es mal testweise eingebaut